packages/bench/src/parallel-babel-plugin/impl.ts

```diff
@@ -52,3 +56,11 @@ export const babelPlugin = (): Plugin => {
     },
   };
 };
+
+const impl: ParallelPluginImplementation = defineParallelPluginImplementation(
+  (_options, _context) => {
+    return babelPlugin();
+  },
+);
+
+export default impl;
```

Comment:
This default export is referenced by 
https://github.com/rolldown/rolldown/blob/ec28db0817ce37b22538f4e606abe60debd38e9d/packages/bench/src/parallel-babel-plugin/index.ts#L5

I added to `knip.jsonc` as an entry, but it would be nice to write this in `src/parallel-babel-plugin/index.ts` directly. Is there a way to declare an entry there? I thinking of a comment like `// @knip-entry ./impl.js`.

Reply:
I should add support for this in Knip, a literal `import.meta.dirname` should be easy (but we will not have it yet for this pull request). Knip doesn't have support for comments such as you describe.